validator/remote/build.gradle

```diff
@@ -7,6 +7,7 @@ dependencies {
   implementation project(':infrastructure:http')
   implementation project(':infrastructure:logging')
   implementation project(':infrastructure:metrics')
+  implementation project(':infrastructure:restapi')
```

Comment:
hmm, slightly unfortunate to need to depend on `:infrastructure:restapi` here rather than just `:infrastructure:json`.  Ideally we'd only depend on restapi when we're actually implementing a restapi (ie it's the server component and we can use type definitions in a client without it).  Not sure how feasible that is though.

Reply:
doesnt look like we need it this PR, removed.